### DISCUSSION TITLE
How to update php.ini settings

### DISCUSSION THREAD

**Discussion by abhi-conceptlz** [beyondcode/herd-community#179] [Q&A] [answered]
<img width="1440" alt="Screenshot 2023-09-18 at 10 36 07 AM" src="https://github.com/beyondcode/herd-community/assets/143715324/d90b8d5e-0b95-4e18-8f3c-1562867aef26">

This is a php.ini file, now I want to update the upload max file size settings and other settings
Please help me how to update these.
Thanks

**Comment by TheAbhishekIN** [comment, marked as answer]
got it 
Just to update the variable in this file
<img width="936" alt="Screenshot 2023-09-18 at 1 33 45 PM" src="https://github.com/beyondcode/herd-community/assets/52767864/08066f01-e4ae-41b9-b4c5-efbcf76937d4">
